### DISCUSSION TITLE
What is 'devland' means?

### DISCUSSION THREAD

**Discussion by luxi78** [quasarframework/quasar#10871] [General - Components / Directives / etc] [answered]
Hi,
`devland` appears in quasar doc and some discussion threads without any description.
Could anyone explain what it means? 

thanks!

**Comment by metalsadman** [comment, marked as answer]
it pertains to you, the developer, your environment, choices etc...

**Reply by luxi78** [reply]
> it pertains to you, the developer, your environment, choices etc...

got it , thanks
